### PR TITLE
Allow re-publishing already published course-runs

### PR DESCRIPTION
**Pull request by inventhouse**

### CODE REVIEW

File: course_discovery/apps/course_metadata/utils.py [hunk 1]
```diff
@@ -53,20 +53,19 @@ def set_official_state(obj, model, attrs=None):
     """
```
Comment (dianakhuang) [inline course_discovery/apps/course_metadata/utils.py:53]: I think we should update the docstring with this new behavior.

**Review by dianakhuang — APPROVED** [review]
Just one minor comment, but this looks good otherwise.

File: course_discovery/apps/api/v1/tests/test_views/test_course_runs.py [hunk 2]
```diff
@@ -289,7 +289,7 @@ def test_partial_update_no_studio_url(self):
         with mock.patch('course_discovery.apps.api.v1.views.course_runs.log.info') as mock_logger:
             response = self.client.patch(url, {}, format='json')
 
-        self.assertEqual(response.status_code, 200)
+        assert response.status_code == 200, "Status {}: {}".format(response.status_code, response.content)
```
Comment (mikix) [inline course_discovery/apps/api/v1/tests/test_views/test_course_runs.py]: assertEqual can also take a third argument as the message.

**Review by mikix — APPROVED** [review]
Seems reasonable! Thanks!

**Pull request merged**